src/program.cpp

```diff
@@ -561,6 +569,7 @@ bool save_string_to_file(const std::string& fname, const std::string& text) {
     return ofile.good();
 }
 
+#ifndef CLSPV_ONLINE_COMPILER
```

Comment:
Why do you care about stripping out a function if it's not used? Did you get a warning?

Reply:
Yes I was getting a warning and also I was attempting to match the similar approach used for the save_string_to_file method above.